src/lib/utils/contributors.ts

```diff
@@ -0,0 +1,47 @@
+import { join } from "path"
+
+import type { CommitHistory, FileContributor, Lang, Layout } from "@/lib/types"
+
+import { DEFAULT_LOCALE } from "@/lib/constants"
+
+import {
+  convertToFileContributorFromCrowdin,
+  getCrowdinContributors,
+} from "./crowdin"
+import { getLastModifiedDate } from "./gh"
+
+import { fetchAndCacheGitContributors } from "@/lib/api/fetchGitHistory"
+
+export const getFileContributorInfo = async (
+  mdDir: string,
+  mdPath: string,
+  slug: string,
+  locale: string,
+  fileLang: string,
+  layout: Layout,
```

Comment:
I see we use this arg to determine whether or not we want to fetch the crowdin contributors. Couldn't we deduce this from the slug? `slug.includes("/developers/")`